acceptance/tests/lookup/configuration_interpolation.rb

```diff
@@ -5,19 +5,21 @@
   app_type        = File.basename(__FILE__, '.*')
   tmp_environment = mk_tmp_environment_with_teardown(master, app_type)
   fq_tmp_environmentpath  = "#{environmentpath}/#{tmp_environment}"
+  master_confdir = master.puppet('master')['confdir']
+
+  step "backup hiera config file" do
+    on master, "cp #{master_confdir}/hiera.yaml #{master_confdir}/hiera.bak}"
```

Comment:
Oh, my restore checks for an existing backup (duh). Not sure that actually makes sense since the first step is to make that backup. How do you feel about making the restore unconditional (since if the backup broke, further tests will likely fail)